documentation/docs/verifiable_credentials/overview.md

```diff
@@ -1,46 +1,49 @@
+**Verifier:** An entity which checks if the VC a holder presents is legitimate – The border agent (the _Verifier_) trusts the government (the _Issuer_) which issued Alice her passport and validates that Alice (the _Subject_) is a citizen.
 
 :::note
 
-See the [Verifiable Credentials Data Model 1.0 Specification](https://w3c.github.io/vc-data-model/) for more information.
+For more information, you can check out the [Verifiable Credentials Data Model 1.0 Specification page](https://w3c.github.io/vc-data-model/).
 
 :::
 
 ### Verifiable Credentials in IOTA
 
-In the IOTA Identity framework, instead of a physical passport being given to Alice with the passport information being written into a centralized database owned by the government, Alice receives a digital verifiable credential, and the information required for verification in the future is written to the Tangle.
+In the IOTA Idenitity framework, Alice would receive a digitally verifiable credential rather than a physical passport. Rather than the passport information being written into a centralized database owned by the government for verfication, the digitial credential is (in the future) written to the Tangle.
 
-At a high level, the creation and verification of a VC on IOTA works as follows:
+The first step in the creation and verification of a VC is to create a verifiable credential. This requires the subject and issuer (Alice and the government, respectively) to have DIDs published to the Tangle with a set of statements being asserted (that Alice has a passport). The issuer signs the credential with their private key and publishes the public key to the Tangle. In the future, a proof can be used by the verifiers to validate the authenticity of the credentials using the issuer's public key.
 
-The first step is to create a verifiable credential which requires the subject (Alice) and issuer (the government) to have DIDs published to the Tangle, and a set of statements being asserted (that Alice has a passport). The issuer signs the credential with their private key and publishes the public key to the Tangle. In the future, a proof can be used to assert the authenticity of the credentials by verifiers using the issuer's public key.
-
-After the credential is published to the tangle, Validation is performed by looking up the issuer's key on the Tangle, the holder proving ownership of their DID to the verifier (evidence), and validating that the credential has indeed been signed by the issuing party.
+After the credential is published to the Tangle, validations occur:
+ 
+ - By looking up the issuer's key on the Tangle. 
+ - The holder proving ownership of their DID to the verifier (evidence). 
+ - Validating that the credential has been signed by the issuing party.
```

Comment:
It doesn't really read like a sentence tbh. Also it is not clear to me who makes validation occur. "Anyone can verify the credentials by looking up the key, checking the holder's DID, and the signature of the issuing party".

"Validations occur by validating" is also kinda awkward.